charmcraft.yaml

```
@@ -0,0 +1,10 @@
+# Learn more about charmcraft.yaml configuration at:
+# https://juju.is/docs/sdk/charmcraft-config
+type: "charm"
+bases:
+  - build-on:
+      - name: "ubuntu"
+        channel: "22.04"
+    run-on:
+      - name: "ubuntu"
+        channel: "22.04"
```

Nothing major. You are just missing a newline at the end of this file.

```suggestion
        channel: "22.04"

```

---

Updated!